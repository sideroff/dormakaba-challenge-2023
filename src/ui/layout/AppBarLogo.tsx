import Link from 'next/link';
import Image from 'next/image';
import Box from '@mui/material/Box';

export function AppBarLogo() {
  return (
    <Box sx={{ display: { xs: 'none', md: 'flex' }, mr: 2 }}>
      <Link href="/">
        <Image
          src="/logo.png"
          priority
          alt="dormakaba logo"
          width="269"
          height="30"
        />
      </Link>
    </Box>
  );
}
