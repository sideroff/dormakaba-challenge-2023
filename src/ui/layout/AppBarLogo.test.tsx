import { render } from '@testing-library/react';
import { AppBarLogo } from './AppBarLogo';

describe('AppBarLogo', () => {
  it('should render correctly', () => {
    const { container } = render(<AppBarLogo />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
