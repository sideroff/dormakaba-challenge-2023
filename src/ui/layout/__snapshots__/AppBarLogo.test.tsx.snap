// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBarLogo should render correctly 1`] = `
<div
  class="MuiBox-root css-8eu3pw"
>
  <a
    href="/"
  >
    <img
      alt="dormakaba logo"
      data-nimg="1"
      decoding="async"
      height="30"
      src="/_next/image?url=%2Flogo.png&w=640&q=75"
      srcset="/_next/image?url=%2Flogo.png&w=384&q=75 1x, /_next/image?url=%2Flogo.png&w=640&q=75 2x"
      style="color: transparent;"
      width="269"
    />
  </a>
</div>
`;
